helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -318,3 +338,39 @@ def action_do_send_email(self):
             "target": "new",
             "context": ctx,
         }
+
+    # ---------------------------------------------------
+    # Portal
+    # ---------------------------------------------------
+
+    def _compute_access_url(self):
+        super(HelpdeskTicket, self)._compute_access_url()
```

Comment:
Why calling super?

Reply:
Because it inherited from `portal.mixin`